mp/src/game/client/momentum/mom_map_cache.cpp

```diff
@@ -374,6 +375,31 @@ void CMapCache::SetMapGamemode(const char *pMapName /* = nullptr*/)
     if (!pMapName)
         pMapName = MapName();
 
+    if (Q_strcmp(mom_gamemode_override.GetString(), "0") && Q_strcmp(mom_gamemode_override.GetString(), ""))
```

Comment:
I'm not a fan of string checking this, I think it's best if it stays just integer-based. Simplifies the logic.

```suggestion
    if (mom_gamemode_override.GetInt() > GAMEMODE_UNKNOWN)
```